charts/spark-operator-chart/templates/pdb.yaml

```
@@ -0,0 +1,13 @@
+{{- if $.Values.pdb.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "spark-operator.fullname" . }}-pdb
+  labels:
+    {{- include "spark-operator.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "spark-operator.selectorLabels" . | nindent 6 }}
+  minAvailable: {{ $.Values.pdb.minAvailable }}
+{{- end }}
```

Cloud you rename this file to `poddisruptionbudget.yaml` and the test file to `poddisruptionbudget_test.yaml` respectively.

Besides, you can add a condition to test whether replica count is greater than 1, since this is needed by the feature.

```suggestion
{{- if .Values.podDisruptionBudget.enable }}
{{- if (gt .ReplicaCount 1) }}
apiVersion: policy/v1
kind: PodDisruptionBudget
metadata:
  name: {{ include "spark-operator.fullname" . }}-pdb
  labels:
    {{- include "spark-operator.labels" . | nindent 4 }}
spec:
  selector:
    matchLabels:
      {{- include "spark-operator.selectorLabels" . | nindent 6 }}
  minAvailable: {{ .Values.podDisruptionBudget.minAvailable }}
{{- else }}
{{- fail "replicaCount must be greater than 1 to enable pod disruption budget" }}
{{- end }}
{{- end }}
```

---

done